### ISSUE TITLE
Ability to export and load FDX files

### ISSUE THREAD

**Issue by Cerabbite** [Lycoon/scriptio#49]
It would be nice if you were able to export your script besides fountain also as FDX.
The same would go for loading FDX files

**Comment by Lycoon** [comment]
Absolutely, it is a high priority feature I am currently investigating on.
I'm planning to add conversion to other file formats such as `.fdx`, `.celtx`, etc.

**Comment by Lycoon** [comment]
FYI @Cerabbite: Final Draft format is now supported for exports! 🎉

The version is currently live on scriptio.app if you wanna try it out.
The import is yet to be done, and other formats will follow. As these formats are proprietary, it will take me few iterations to tune my algorithms to have the best possible quality data on export/import.

**Comment by Lycoon** [comment]
Final Draft is available too as of right now!
